wpilibNewCommands/src/main/native/cpp/frc2/command/CommandScheduler.cpp

```diff
@@ -288,6 +288,18 @@ void CommandScheduler::UnregisterSubsystem(
   }
 }
 
+const std::vector<Subsystem*> CommandScheduler::GetAllRegisteredSubsystems() {
+  std::vector<frc2::Subsystem*> subsystems;
+  subsystems.reserve(m_impl->subsystems.size());
+  for (wpi::DenseMap<frc2::Subsystem*, std::unique_ptr<Command>>::iterator it =
```

Comment:
Since the type is pretty clear, just use `auto`:
```suggestion
  for (auto it =
```